v2/fleet/datadog-control.service

```diff
@@ -0,0 +1,26 @@
+# https://raw.githubusercontent.com/DataDog/docker-dd-agent/master/dd-agent.service
+# make sure the etcd /ddapikey is set!
+[Unit]
+Description=Datadog Agent for Mesos, Marathon & Zookeeper
+After=docker.service
+
+[Service]
+EnvironmentFile=/etc/environment
+User=core
+Restart=always
+ExecStartPre=/usr/bin/docker pull behance/docker-dd-agent-mesos:latest
+ExecStartPre=-/usr/bin/docker stop dd-agent-mesos
+ExecStartPre=-/usr/bin/docker rm -f dd-agent-mesos
+TimeoutStartSec=0
+ExecStartPre=-/usr/bin/docker kill dd-agent-mesos
+ExecStartPre=-/usr/bin/docker rm dd-agent-mesos
+ExecStart=/usr/bin/bash -c \
+"if [[ -f /etc/profile.d/etcdctl.sh ]]; then source /etc/profile.d/etcdctl.sh;fi && sudo /usr/bin/docker run --name dd-agent-mesos -h `hostname` \
+-e API_KEY=`etcdctl get /ddapikey` \
+-e MESOS_HOST \
```

Comment:
Nope, because I'm monitoring mesos, zookeeper and marathon, I append the ports in the [entrypoint script](https://github.com/behance/docker-dd-agent-mesos/blob/master/entrypoint.sh#L11). Not as flexible should we decide to change the ports, but it's easy enough to redeploy the container at this point.